### ISSUE TITLE
On sign out redirect to a deconnexion page

### ISSUE THREAD

**Issue by pyphilia** [graasp/graasp-compose#274]

**Comment by mayankdp** [comment]
Hii, I want to contribute to this open-source issue. I am very interested in working on this issue. So can I work on this issue?

**Comment by pyphilia** [comment]
@mayankdp of course! Thank you for your interest. Currently the necessary backend is not available. But it will really soon be accessible. I think you can wait for it. Or try to mock some key endpoints, but they are numerous. But it shows that it might be interesting to have a contained mock backend to work on this project as standalone, I'll create a dedicated issue.

**Comment by mayankdp** [comment]
Sorry for the late reply. Yes, I'll wait for the backend to be available. In the meantime, I'll run the project.
